lib/spider/configs/prompts/icl-3-shot.yaml

```diff
@@ -0,0 +1,21 @@
+defaults:
+  - default
+
+name: "icl-3-shot"
```

Comment:
The name is super weird & a bit confusing: after some thoughts I understand what it is but at first glance I wondered what's the relation between this & `icl-3` 
You could use a different synthax, like `icl_3_shot-X` (where X can be a number) to show the difference, wdyt? (both in the `name` here & the file name)